apps/oc_chef_wm/src/oc_chef_wm_cookbook_artifacts.erl

```diff
@@ -0,0 +1,107 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
+%% ex: ts=4 sw=4 et
+%% @author Jean Rouge <jean@chef.io>
+%% Copyright 2013-2015 Chef Software, Inc. All Rights Reserved.
+
+-module(oc_chef_wm_cookbook_artifacts).
+
+-include("../../include/oc_chef_wm.hrl").
+
+-behaviour(chef_wm).
+
+%% Webmachine resource callbacks
+-mixin([{oc_chef_wm_base, [content_types_accepted/2,
+                           content_types_provided/2,
+                           finish_request/2,
+                           malformed_request/2,
+                           ping/2,
+                           forbidden/2,
+                           is_authorized/2,
+                           service_available/2,
+                           validate_request/3]}]).
+
+-export([auth_info/2,
+         init_resource_state/1,
+         init/1,
+         request_type/0,
+         allowed_methods/2,
+         to_json/2,
+         malformed_request_message/3]).
+
+init(Config) ->
+    oc_chef_wm_base:init(?MODULE, Config).
+
+init_resource_state(_Config) ->
+    {ok, #cookbook_artifacts_state{}}.
+
+request_type() ->
+    "named_cookbook_artifact".
+
+allowed_methods(Req, State) ->
+    {['GET'], Req, State}.
+
+auth_info(Req, #base_state{organization_guid = OrgId,
+                           chef_db_context = DbContext,
+                           resource_args = single_artifact} = State) ->
+    Name = wrq:path_info(name, Req),
+    BaseRec = #oc_chef_cookbook_artifact{org_id = OrgId,
+                                         name = Name},
+    case chef_db:fetch(BaseRec, DbContext) of
+        not_found ->
+            fail_with(Req, State, 404, "not found", cookbook_artifact_not_found);
+        forbidden ->
```

Comment:
I traced back through the usage, and I don't see any way the call to chef_db:delete would return forbidden? Same applies in some existing/modified captures of this result elsewhere in the WM resources